src/doc/trpl/looping.md

```diff
@@ -54,7 +54,7 @@ The other kind of looping construct in Rust is the `while` loop. It looks like
 this:
 
 ```{rust}
-let mut x = 5u;       // mut x: uint
+let mut x = 5u32;       // mut x: u32
```

Comment:
Strictly speaking, you can put no suffix and get the same code (well, it would use i32), but I'm not sure if that effects the meaning steve was going for.